probe-rs/src/debug/mod.rs

```diff
@@ -328,6 +339,9 @@ impl DebugInfo {
             frame_section,
             locations_section,
             address_section,
+            // The minimum instruction size in bytes.
+            // TODO: Do this programatically, based on architecture.
```

Comment:
I think this slipped :)

Reply:
No, I think this is a legitimate question ... I updated the comment to clarify my intent.